src/content/docs/specs/versioning/get-versions.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: Get supported versions
+description: Query a server for its supported versions
+sidebar:
+  order: 2
+  badge:
+   text: GET
+   variant: success
+---
+
+import CoreAction from "@partials/_core-action.mdx";
+
+<CoreAction />
+
+```http title="Endpoint"
+GET /v1/versions
```

Comment:
The versions endpoint should be the only one not behind `/v1/` so that clients can connect to it without knowing supported versions.
```suggestion
GET /versions
```